orchagent/fgnhgorch.cpp

```diff
@@ -1227,6 +1405,15 @@ bool FgNhgOrch::setFgNhg(sai_object_id_t vrf_id, const IpPrefix &ipPrefix, const
         return true;
     }
 
+    if (!fgNhgEntry)
+    {
+        SWSS_LOG_INFO("fgNhgOrch got a route addition %s:%s with %s",
```

Comment:
This should never happen because routeorch checks for ```if (isRouteFineGrained)``` first, and I am not aware of any case where this has happened before. Makes sense to have this check for defensive programming and if this ever happens it is an error condition, can you change this log to error?